services/klatchat_observer/controller.py

```diff
@@ -419,13 +426,17 @@ def get_neon_request_structure(msg_data: dict):
         return request_dict
 
     def __handle_neon_recipient(self, recipient_data: dict, msg_data: dict):
+        """
+        Handle a chat message intended for Neon.
+        """
         msg_data.setdefault("message_body", msg_data.pop("messageText", ""))
         msg_data.setdefault("message_id", msg_data.pop("messageID", ""))
         recipient_data.setdefault("context", {})
         pattern = re.compile("Neon", re.IGNORECASE)
         msg_data["message_body"] = (
-            pattern.sub("", msg_data["message_body"], 1).strip("<>@,.:|- ").capitalize()
+            pattern.sub("", msg_data["message_body"], 1).strip("<>@,.:|- \n")
         )
+        # This is really referencing an MQ endpoint (i.e. stt, tts), not a skill
```

Comment:
It used to be referencing a skill by dedicated endpoint, but currently it looks like it does not quite operation for Neon

Reply:
I'm actually not sure when I left this note; I think I had some pending changes locally from stepping through the code before.. I'll remove these changes that aren't relevant to this PR